# config.yaml
odom_topic: "/robot/dlo/odom_node/odom"
robot_frame: "robot/base_link"
trajectory_topic: "trajectory"
frame_id: "map"
pathtosavelandmarkcsv: "config/landmark.csv"
pathtoloadlandmarkcsv: "config/landmark.csv"
savetaglocation: true
usepriortagtable: false
batch_optimisation: true
total_tags: 626
transformation_search_range: 0.1
add2graph_threshold: 0.2
timewindow: 50
maxfactors: 400
useprunebytime: false
useprunebysize: false

noise_models:
  odometry:
    - 0.0001
    - 0.01
    - 0.0001
  prior:
    - 0.1
    - 0.3
    - 0.1
  bearing_range:
    - 0.1
    - 0.8
  point:
    - 10
    - 10
